### PR TITLE
[PLAYER-4004] Fix for misaligned Player Control bar

### PR DESCRIPTION
**Pull request by SergeyBicarte**
Added correct calculation for size of OoyalaSkinLayout for all cases (toggle full screen/orientation changing)

### CODE REVIEW

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java [hunk 1]
```diff
@@ -159,36 +165,29 @@ void setFullscreen(boolean fullscreen) {
...
-      } else {
-        //TODO: Android 16 and below will show a poor fullscreen experience right now
-        // It will not fill the screen where the navigation bar is supposed to be
-        windowManager.getDefaultDisplay().getMetrics(displayMetrics);
-      }
-      layoutParams.width = displayMetrics.widthPixels;
       layoutParams.height = displayMetrics.heightPixels;
-      bringToFront();
+    } else {
+      layoutParams.height = displayMetrics.heightPixels-statusBarHeight;
```
Comment (ZadorozhnyiSemen) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: Just missing spaces around `-`. Everything else looks good

Comment (ZadorozhnyiSemen) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: Just missing spaces around `-`. Everything else looks good

**Review by ZadorozhnyiSemen — CHANGES_REQUESTED** [review]

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java (hunk 1 above)
Comment (SergeyBicarte) [reply sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java]: Done

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java [hunk 2]
```diff
@@ -159,36 +165,29 @@ void setFullscreen(boolean fullscreen) {
...
         return;
     }
-    boolean changed = this.fullscreen != fullscreen;
-    ViewGroup.LayoutParams layoutParams = getLayoutParams();
-
 
-    if (changed) {
-      if (fullscreen) {
-        //Store the source size of the layout
-        sourceWidth = layoutParams.width;
```
Comment (brol1dev) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java:169]: do you no longer need the sourceWidth and height? Remember we needed it to know the original width and height before going into fullscreen mode.

**Review by brol1dev — APPROVED** [review]
not quite sure if it will work but we can give it a try.

**Comment by brol1dev** [comment]
however this fix is for the next release (4.41) so holding the merge until we release the current one.

**Comment by SergeyBicarte** [comment]
I am changed that fix, no need size calculation at all, just added forceLayout() calls at point when size really changed (after orientation changing and after hide/show bars), this also fixing  [PLAYER-3034](https://jira.corp.ooyala.com:8443/browse/PLAYER-3034)

**Review by SZadorozhnyiEpam — APPROVED** [review]

**Pull request merged**